### PR TITLE
Erin burntarea

### PR DESCRIPTION
**Pull request by erin-telfer**

### CODE REVIEW

File: dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            "dynamic": True,
+            "native_crs": "EPSG:3577",
+            "native_resolution": [10.0, 10.0],
+            "image_processing": {
+                "extent_mask_func": "datacube_ows.ogc_utils.mask_by_val",
+                "always_fetch_bands": [],
+                "manual_merge": False,
+            },
+            "wcs": {
+                "default_bands": [
```
Comment (pindge) [inline dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py:189]: the items here are suppose to be `bands` not styles

File: dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        {"value": -0.4, "color": "#CEAC0E"},
+        {"value": -0.2, "color": "#E5D609"},
+        {"value": 0.0, "color": "#FFFF0C"},
+        {"value": 0.2, "color": "#C3DE09"},
+        {"value": 0.4, "color": "#88B808"},
+        {"value": 0.6, "color": "#529400"},
+        {"value": 0.8, "color": "#237100"},
+        {"value": 1.0, "color": "#114D04"}, ],
+    "legend": {  # need this
+        "url": "", },
```
Comment (pindge) [inline dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py:86]: can we push this closing `},` to the new line so its more readable?

File: dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    "needed_bands": ["delta_nbr"],
+    "index_function": {
+        "function": "datacube_ows.band_utils.single_band",
+        "mapped_bands": True,
+        "kwargs": {
+            "band": "delta_nbr",
+        },
+    },
+    "range": [-1, 1],
+    "mpl_ramp": "PuOr",  # need to ask how to reverse colour ramp
```
Comment (pindge) [inline dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py:119]: add  `_r` to the end wil be reverse, be careful not to have any space

**Review by pindge — CHANGES_REQUESTED** [review]

**Comment by pindge** [comment]
@erin-telfer you also need to update inventory json https://github.com/GeoscienceAustralia/dea-config/blob/master/dev/services/wms/inventory.json

and ows_root_cfg.py https://github.com/GeoscienceAustralia/dea-config/blob/master/dev/services/wms/ows_refactored/ows_root_cfg.py

File: dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py (hunk 3 above)
Comment (erin-telfer) [reply dev/services/wms/ows_refactored/burntarea/ows_provisional_ba_cfg.py:119]: To clarify, this is the correct text?
"mpl_ramp": "PuOr_r",

File: dev/services/wms/inventory.json [hunk 4]
```diff
@@ -1214,7 +1214,19 @@
...
+                "ga_s2_ba_provisional_3"
+            ],
+            "styles_count": 4,
+            "styles_list": [
+                "style_dnbr",
+                "style_dnbr_classes",
+                "style_dbsi",
+                "style_dndvi",
+            ]
+        },
```
Comment (pindge) [inline dev/services/wms/inventory.json:1229]: json doesnt like trailin `,` at the end

File: dev/services/wms/inventory.json [hunk 5]
```diff
@@ -1214,7 +1214,19 @@
...
+        {
+            "product": [
+                "ga_s2_ba_provisional_3"
+            ],
+            "styles_count": 4,
+            "styles_list": [
+                "style_dnbr",
+                "style_dnbr_classes",
+                "style_dbsi",
+                "style_dndvi",
```
Comment (pindge) [inline dev/services/wms/inventory.json:1227]: so is this `,`

**Review by pindge — CHANGES_REQUESTED** [review]

**Review by erin-telfer — COMMENTED** [review]
-done

**Review by pindge — APPROVED** [review]

**Pull request merged**